internal/services/automation/automation_account_resource.go

```diff
@@ -71,10 +73,104 @@ func resourceAutomationAccount() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeString,
 				Computed: true,
 			},
+
+			"identity": {
```

Comment:
hey @damienpontifex 

Thanks for this PR - apologies for the delayed review on this one - I'm currently working through the provider to make the `identity` blocks consistent as a part of #15187, as such I hope you don't mind but I'm going to push a couple of commits to update this to [use the Common Schema for Identity](https://github.com/hashicorp/go-azure-helpers/tree/main/resourcemanager/commonschema) so that this is consistent with that.

Thanks!

Reply:
I would never mind @tombuildsstuff :) In this together to make things awesome, so I appreciate your input to make this consistent and shippable.
Thanks for the commits and ongoing work here 👏🏻